code/mod_table/mod_table.cpp

```diff
@@ -48,6 +48,7 @@ bool Enable_external_default_scripts;
 int Default_detail_level;
 bool Full_color_head_anis;
 bool Dont_automatically_select_turret_when_targeting_ship;
+bool Automatically_select_subsystem_underreticle_when_targeting_ship;
```

Comment:
Very pedantic, but let's put an underscore between `under` and `reticle` for the consistency.